.vscode/c_cpp_properties.json

```diff
@@ -1,15 +1,25 @@
 {
+    "env": {
+        "projectPath": "/home/matkozyra52/Documents/studia/sensory/projekt/Sensors_in_embedded_systems_software"
+      },
     "configurations": [
         {
             "name": "Linux",
             "includePath": [
                 "${default}",
-                "/home/roiz/Codes/Sensors_in_embedded_systems_software/includes",
-                "/home/roiz/Codes/Sensors_in_embedded_systems_software/Drivers/STM32F1xx_HAL_Driver/Inc",
-                "/home/roiz/Codes/Sensors_in_embedded_systems_software/Drivers/Config/Include",
-                "/home/roiz/Codes/Sensors_in_embedded_systems_software/Drivers/stm32f103/CMSIS/Device/ST/STM32F1xx/Include"
+                "${projectPath}/includes",
+                "${projectPath}/stm32f429/CMSIS/Include",
+                "${projectPath}/Drivers/STM32F4xx_HAL_Driver/Inc",
+                "${projectPath}/Drivers/stm32f429/CMSIS/Device/ST/STM32F4xx/Include",
+                "${projectPath}/Drivers/Config/Include",
+                "${projectPath}/Drivers/stm32f103/CMSIS/Include",
+                "${projectPath}/Drivers/STM32F1xx_HAL_Driver/Inc",
+                "${projectPath}/Drivers/stm32f103/CMSIS/Device/ST/STM32F1xx/Include",
+                "${projectPath}/Drivers/Config/Include"
+            ],
+            "defines": [
+                "STM32F4"
```

Comment:
w takim razie, to też może niech będzie w env? Żeby nie grzebać po makefile, tylko zmieniać wszystko tam